tests/typescript/conformance/types/mappedType/mappedType.ts

```diff
@@ -1,3 +1,3 @@
 type Keys = 'option1' | 'option2';
 type Flags = { [K in Keys]: boolean };
-
+type Anys = { [K in Keys] };
```

Comment:
I'm not familiar with the background, but I think it is better to create new directory named `/tests/typescript_mapped_type` than adding tests to `/tests/typescripts/conformance`.
(I was previously pointed on a similar thing from @thron0. See https://github.com/prettier/prettier/pull/6604#issuecomment-538934227)

Reply:
It makes sense. I'll try it.

Reply:
wait, should I create new tests at `/tests/typescript_mapped_type`  while leaving `/tests/typescripts/conformance/mappedType.ts`? Or move `/tests/typescripts/conformance/mappedType.ts` to `/tests/typescript_mapped_type`?

Reply:
I understood the problem by reading the above conversation so I'm going to create new dir  while keeping `/conformance/mappedType.ts` before :bow: